frontend/src/modules/indices/reducers.js

```diff
@@ -77,41 +36,15 @@ export const indices = (
         case INDICES.LIST.ERROR:
             return { ...state, isFetching: false, error: action.error, };
 
-        case INDICES.LIST_TABLE.REQUEST:
-            return { ...state, isFetching: true, };
-        case INDICES.LIST_TABLE.RECEIVE: {
-            const totalCount = action.data.count;
-            const hasChanged = state.totalCount !== action.data.count;
-            const currentItems = hasChanged ? [] : state.items;
-            const results = action.data.results.map(preprocess)
-            const newItemsByID = map(
-                s => ({ ...s, container: s.container }),
-                indexByID(results)
-            );
-            const itemsByID = merge(state.itemsByID, [], newItemsByID);
-            const itemsID = action.data.results.map(r => r.id)
-            const items = mergeArray(currentItems, action.meta.offset, itemsID)
-            return {
-                ...state,
-                itemsByID,
-                items,
-                totalCount,
-                page: action.meta,
-                isFetching: false,
-                error: undefined,
-            };
-        }
-        case INDICES.LIST_TABLE.ERROR:
-            return { ...state, isFetching: false, error: action.error, };
-        default:
-            return state;
-
         case INDICES.VALIDATE.REQUEST:
             return { ...state, error: undefined, isFetching: true };
         case INDICES.VALIDATE.RECEIVE:
             return {...state, isFetching: false };
         case INDICES.VALIDATE.ERROR:
             return { ...state, error: action.error, isFetching: false };
+
+        default:
+            return state
```

Comment:
Calling a unknown reducers do not generate an error ?

Reply:
Reducers receive every action that gets dispatched. If the action isn't one that the reducer is interested in then it must return the unmodified state. I'm not sure how that original reducer worked without causing some kind of error or crash.

Reply:
The default is likely set in combineReducers ... looks like the expected value in any case.